docs/_static/C++/CMakeLists.txt

```diff
@@ -0,0 +1,129 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(TestVisualizer)
+
+find_package(PkgConfig QUIET)
+find_package(OpenGL    QUIET)
+find_package(pybind11  QUIET)
+find_package(OpenMP    QUIET)
+find_package(Open3D    QUIET HINTS "~/open3d_install/lib/CMake")
+
+if (PKGCONFIG_FOUND)
+    pkg_search_module(EIGEN3    eigen3>=3.2.7   QUIET)
+    pkg_search_module(JPEG      libjpeg         QUIET)
+    pkg_search_module(GLFW      glfw3           QUIET)
+    pkg_search_module(GLEW      glew            QUIET)
+    pkg_search_module(JSONCPP   jsoncpp>=1.7.0  QUIET)
+    pkg_search_module(PNG       libpng>=1.6.0   QUIET)
+endif (PKGCONFIG_FOUND)
+
+option(Open3D_USE_NATIVE_DEPENDENCY_BUILD "Prioritize Found dependency build" ON)
```

Comment:
Remove ``Open3D_USE_NATIVE_DEPENDENCY_BUILD``, it is not used any more.

Reply:
done.